modules/fts/pages/fts-searching-from-the-ui.adoc

```diff
@@ -83,6 +94,97 @@ Once the new index has been built, it supports Full Text Searches performed by a
 
 The *indexing progress* is determined as index_doc_count / source_doc_count. While index_doc_count is retrieved from search endpoint, source_doc_count is retrieved from a KV endpoint. In the event where one or more of the nodes in the cluster running data service go down and/or are failed over, *indexing progress* may show a value > 100% as the source_doc_count for the bucket would be missing some active partitions.
 
+[#fts-quick-index]
+=== Quick Index
+To create a quick index, left-click on the *QUICK INDEX* button, towards the right-hand side:
+
+[#fts_quick_index_button]
+image::fts-quick-index-button.png[,120,align=left]
+
+The *QUICK INDEX* screen appears:
+
+[#fts_quick_index_screen]
+image::fts-quick-index-screen.png[,640,align=left]
+
+To define a basic index on which Full Text Search can be performed, begin by entering a unique name for the index into the *Index Name* field, at the upper-left: for example, `travel-sample-index`. (Note that only alphanumeric characters, hyphens, and underscores are allowed for index names.
+Note also that the first character of the name must be an alphabetic character.) Then, use the pull-down menu provided for the *Keyspace* field, at the upper-right, to specify as follows:
+
+bucket: `travel-sample`
+
+scope: `inventory`
+
+collection: `hotel`
+
+[#fts_quick_index_name_and_bucket]
+image::fts-quick-index-name-and-bucket.png[,640,align=left]
+
+Once the Keyspace is defined, the document gets selected randomly from the specified *Keyspace* fields and displayed at the bottom side of the Index Name field in JSON format.
```

Comment:
We need to add something like -> the user can continue to randomly pick documents until they find a document of their intended type/schema. Reason being,  it is still possible to have multi schema documents with a collection..

Reply:
Done as part of commit 29f1008255ee63b62ee3106e1c9179a4238c6976.